bluebottle/tasks/models.py

```diff
@@ -371,6 +371,10 @@ class Analytics:
             'task_id': 'task_member.task.id'
         }
 
+        def extra_fields(self, obj, created):
+            # Force the time_spent to an int.
+            return {'hours': int(obj.task_member.time_spent)}
```

Comment:
I just carried it over from how it was done for TaskMember analytics. time_spent is a PositiveSmallIntegerField.